source/module_base/test/tool_threading_test.cpp

```diff
@@ -54,3 +56,47 @@ TEST(tool_threading, TASK_DIST_1D_test_longlong)
     EXPECT_EQ(st,6);
     EXPECT_EQ(le,6);
 }
+
+TEST(tool_threading, BLOCK_TASK_DIST_1D_test_int)
+{
+    int nw=1;
+    int iw=1;
+    int nt=16;
+    int st;
+    int le;
+    int bs=1;
+    ModuleBase:: BLOCK_TASK_DIST_1D(nw,iw,nt,bs,st,le);
+    EXPECT_EQ(st,0);
+    EXPECT_EQ(le,16);
+    int nw2=4;
+    ModuleBase:: BLOCK_TASK_DIST_1D(nw2,iw,nt,bs,st,le);
+    EXPECT_EQ(st,4);
+    EXPECT_EQ(le,4);
+    int nw3=3;
+    int nt3=17;
+    ModuleBase:: BLOCK_TASK_DIST_1D(nw3,iw,nt3,bs,st,le);
+    EXPECT_EQ(st,6);
+    EXPECT_EQ(le,6);
+}
+
+TEST(tool_threading, BLOCK_TASK_DIST_1D_test_longlong)
+{
+    long long nw=1;
+    long long iw=1;
+    long long nt=16;
+    long long st;
+    long long le;
+    long long bs=1;
+    ModuleBase:: BLOCK_TASK_DIST_1D(nw,iw,nt,bs,st,le);

```

Comment:
Could you please give short comments on the meaning of the parameters?

Reply:
> Could you please give short comments on the meaning of the parameters?

They are abbreviations for function parameter names.
nw: nworker
iw: iworker
nt: ntask
st: start
le: len
bs:  block_size

Reply:
Adding them into the test file as comments will facilitate future review.